src/server/routes/forwardingRoute.js

```diff
@@ -24,18 +28,30 @@ async function findNBNodeId(nodeId, lang) {
   const response = await fetchWithAccessToken(
     `${baseUrl}/external_ids/${nodeId}`,
   );
+
+  // The nodeId my be a learningpath (return nodeId)
+  if (response.status === 404) {
+    return nodeId;
+  }
+
   const data = await resolveJsonOrRejectWithError(response);
 
   // The nodeId for language nb is the first item in externalIds array.
   return data.externalIds[0];
 }
 
-async function taxonomyLookup(url) {
+async function lookup(url) {
   const baseUrl = apiResourceUrl('/taxonomy/v1/url/mapping');
   const response = await fetchWithAccessToken(`${baseUrl}?url=${url}`);
   return resolveJsonOrRejectWithError(response);
 }
 
+async function resolve(path) {
+  const baseUrl = apiResourceUrl('/taxonomy/v1/url/resolve');
```

Comment:
Shouldnt the `baseUrl` be `/taxonomy/v1/url`?

Reply:
Does'nt really change anything. Can't have a global baseurl since we also make calls to article-api.